samples/package.json

```diff
@@ -17,9 +17,9 @@
     "test": "node -e 'console.log(`no tests`);'"
   },
   "dependencies": {
-    "@google-cloud/container": "0.2.0"
+    "@google-cloud/container": "^0.2.0"
```

Comment:
hmm. I'm not sure about this change since we explicitly want the samples to use exactly the same version of the code

Reply:
As long as we have a nightly test running, this should be fine right?  Pinning to a specific version is dangerous since we don't get security patches.

Reply:
+1 to not pinning.

Reply:
Well, it basically means that samples are not bound to the exact version of the code. 

If we go without pinning here, we might as well need to fix `repo-tools bump` command that writes this version number here (if anyone except me uses it).

Reply:
+1 - I can take a look at that in a bit